MSAL/MSAL.xcodeproj/project.pbxproj

```diff
@@ -1059,7 +1091,7 @@
 		DECE0F292BE3EB010036738C /* MSALNativeAuthInternalError.swift in Sources */ = {isa = PBXBuildFile; fileRef = E206FC5E296D65DE00AF4400 /* MSALNativeAuthInternalError.swift */; };
 		DECE0F312BE3EB0B0036738C /* MSALNativeAuthCacheAccessor.swift in Sources */ = {isa = PBXBuildFile; fileRef = 289E156C2948EB8A006104D9 /* MSALNativeAuthCacheAccessor.swift */; };
 		DECE0F322BE3EB0D0036738C /* MSALNativeAuthCacheInterface.swift in Sources */ = {isa = PBXBuildFile; fileRef = 289E15582948E601006104D9 /* MSALNativeAuthCacheInterface.swift */; };
-		DECE0F332BE3EB0F0036738C /* MSALNativeAuthTokens.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2884855B295DAFD400516492 /* MSALNativeAuthTokens.swift */; };
+		DECE0F332BE3EB0F0036738C /* (null) in Sources */ = {isa = PBXBuildFile; };
```

Comment:
Is this `(null)` expected to be here? Same in line 6549

Reply:
Removed and all tests look still ok